src/components/scripture/reference.ts

```diff
@@ -128,11 +129,11 @@ export const bookCodeToName = (code: string) => {
 export const createRandomScriptureReferences = (): ScriptureRangeInput[] => {
   //To do
   // const book = books[Math.floor(Math.random() * books.length)];
-  const book = books[Math.floor(Math.random() * 2)];
+  const book = books[random(1)];
   // const startChapter = book.chapters[Math.floor(Math.random() * book.chapters.length)];
-  const startChapter = book.chapters[Math.floor(Math.random() * 2)];
+  const startChapter = book.chapters[random(1)];
   // const endChapter = book.chapters[Math.floor(Math.random() * book.chapters.length)];
-  const endChapter = book.chapters[Math.floor(Math.random() * 2)];
+  const endChapter = book.chapters[random(1)];
```

Comment:
That's only 0 or 1. Why not
```suggestion
  const endChapter = book.chapters[random(book.chapters.length - 1)];
```

Reply:
That is what I tried before on my local. Sometimes, invalid verse number error is occurring when selecting the random start and end chapters. I created an issue about it. Need to figure out the reason.

Reply:
Ok sounds good